src/utils/app-api-utils.js

```diff
@@ -98,6 +100,32 @@ export function logout(dispatch) {
   });
 };
 
+export function getDashboardStats(dispatch) {
+  let url = `${apiEndPoint()}/dashboard/stats`;
+  axios.get(
+    url,{
+      headers: {
```

Comment:
Headers can also be moved out to a common function, since this will be used in lot of api calls.